diff --git a/node_modules/@stencil/core/.DS_Store b/node_modules/@stencil/core/.DS_Store
new file mode 100644
index 0000000..429a3d6
Binary files /dev/null and b/node_modules/@stencil/core/.DS_Store differ
diff --git a/node_modules/@stencil/core/internal/stencil-core/index.js b/node_modules/@stencil/core/internal/stencil-core/index.js
index dcb21f9..1496174 100644
--- a/node_modules/@stencil/core/internal/stencil-core/index.js
+++ b/node_modules/@stencil/core/internal/stencil-core/index.js
@@ -1,5 +1,6 @@
 export {
   Build,
+  Env,
   getAssetPath,
   getElement,
   getMode,
