diff --git a/node_modules/@storybook/core/dist/core-server/index.cjs b/node_modules/@storybook/core/dist/core-server/index.cjs
index 4fe721f..83c6d76 100644
--- a/node_modules/@storybook/core/dist/core-server/index.cjs
+++ b/node_modules/@storybook/core/dist/core-server/index.cjs
@@ -257040,9 +257040,9 @@ s unused")).filter(
 i(Nae, "buildDevStandalone");
 
 // src/core-server/standalone.ts
-var YQe = require("node:path");
+var fdPackJson = require("fd-package-json");
 async function mWi(e = {}, t = {}) {
-  let { mode: r = "dev" } = e, n = (0, YQe.dirname)(require.resolve("@storybook/core/package.json")), s = {
+  let { mode: r = "dev" } = e, n = fdPackJson.findPackageSync(__dirname), s = {
     ...e,
     ...t,
     frameworkPresets: [
diff --git a/node_modules/@storybook/core/dist/core-server/index.js b/node_modules/@storybook/core/dist/core-server/index.js
index a4cf5ee..e888222 100644
--- a/node_modules/@storybook/core/dist/core-server/index.js
+++ b/node_modules/@storybook/core/dist/core-server/index.js
@@ -257106,9 +257106,9 @@ s unused")).filter(
 i(Gze, "buildDevStandalone");
 
 // src/core-server/standalone.ts
-import { dirname as $Wi } from "node:path";
+import { findPackageSync } from "fd-package-json";
 async function YWi(e = {}, t = {}) {
-  let { mode: r = "dev" } = e, n = $Wi(Kt.resolve("@storybook/core/package.json")), s = {
+  let { mode: r = "dev" } = e, n = findPackageSync(__dirname), s = {
     ...e,
     ...t,
     frameworkPresets: [
